com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -482,6 +482,7 @@ TextureHandle GetPostprocessUpsampledOutputHandle(RenderGraph renderGraph, strin
                     if (hdCamera.antialiasing == HDAdditionalCameraData.AntialiasingMode.TemporalAntialiasing)
                     {
                         source = DoTemporalAntialiasing(renderGraph, hdCamera, depthBuffer, motionVectors, depthBufferMipChain, source, postDoF: false, "TAA Destination");
+                        RestoreNonjitteredMatrices(renderGraph, hdCamera);
```

Comment:
This looks good, **however** if physically based dof is enabled we have an additional TAA pass which I wonder if it means we should move the "Post TAA" injection point

Reply:
The post TAA point is called before post-process, so I don't think it's a good idea to put it after dof. In any case, I think it will break a lot of stuff if we do that.

About the additional TAA pass, is it done on the whole screen or another buffer?

Reply:
> The post TAA point is called before post-process, so I don't think it's a good idea to put it after dof. In any case, I think it will break a lot of stuff if we do that.

Before post process but after TAA I assume, no? What I mean is that by doing so DoF runs on the full final color buffer after, so it will smudge what the user didn't want to smudge.

Reply:
I guess can be a matter of documentation, but is something we should address _somehow_